python/device/device.py

```diff
@@ -518,9 +580,18 @@ def dispatch_incoming_message_thread(self, worker_thread_info):
                 if thief and thief["pairingId"] == self.pairing_id:
                     cmd = thief["cmd"]
                     if cmd == "pingbackResponse":
+                        print("received {} message with {}".format(cmd, thief["pingbacks"]))
```

Comment:
Why print sometimes, and logger.info other times?

Reply:
oops.  changing to logger.info.